docs/conductor/getting-started.md

```diff
@@ -0,0 +1,88 @@
+# Getting started with Conductor
+##
+
+Conductor will group and schedule automated Dependency Updates on your own Continuous Integration Platform. If the update succeeds, Conductor sends you a pull request to your code hosting platform (GitHub, GitLab, Bitbucket) with the changed composer.lock file and, if necessary, other files modified by Composer plugins or scripts.
+
+To use Conductor:
+
+- [subscribe to the waitlist](http://packagist.com.lo/features/conductor)
+- subscribe to a Private Packagist Subscription (either Cloud or Self-Hosted)
```

Comment:
Wonder if for now, this should say, "once approved, subscribe..." to not give them the impression that they can subscribe and immediately get access

Reply:
yeah, had no nice way to say: "you need a Private Packagist subscription" here. Because looking at the list there are a few using Private Packagist already that were subscribed.

Any ideas?

Reply:
I mean, you don't need a subscription to try it out. Can do so during a trial for instance

Reply:
ah yes. "Subscribe" is missleading, yes